Vision/Vision.bs

```diff
@@ -81,24 +74,23 @@ Boilerplate: omit conformance
 	in the standards we create.
 
 	Technology is not neutral;
-	new technologies enable new actions and new possibilities.
-	We are proud of the good the Web has enabled;
-	we must take the responsibility to use our values
+	new technologies enable new actions and new possibilities, and
+	we must take responsibility 
 	to address the actual impact of our work.
+	The W3C’s Technical Architecture Group's work 
+	to clearly define <a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/">Ethical Web Principles</a> is a strong basis 
+	to improve the ethical integrity of the Web. 
 
-	Our vision is for a World Wide Web that is more inclusive,
```

Comment:
Restructured for flow.

Reply:
This is a bit more than "restructuring for flow". :/

+1 to the restructuring above.
-1 to the sentence linking to EWP. I'm not fundamentally opposed to introducing a cross-reference, but this sentence here is definitely awkward.

Reply:
References to the Ethical Web Principles need to be removed from the vision document for two reasons:

1. The Vision should stand alone and not call out other documents.
2. The EWP is a troubling document. To include the reference would require those issues to be resolved.